app/javascript/scimo/index.js

```diff
@@ -10,10 +10,11 @@ import "./styles/StatesOfMatter.css";
 import "./styles/CellsTissuesOrgans.css";
 import "./styles/lesson_template.css";
 import "./styles/SolarSystem.css";
-
+import "./styles/Diffusion.scss";
 
 import "./scripts/DNA.js";
 import "./scripts/Sound.js";
 import "./scripts/Eye.js";
 import "./scripts/Atoms.js";
 import "./scripts/smoothscroll.js";
+import "./scripts/Diffusion.js";
```

Comment:
This is one way to import the d3 library, but it is more performant to do 

import { scaleLinear } from 'd3-scale’ inside components/lessons/Diffusion.js 

You will also have to run `yarn add d3` in the terminal

Reply:
I tried implementing the code as per your review and doing yarn add d3.  I also tried some other ways of implementing it.  However, unfortunately, import { scaleLinear } from 'd3-scale’ was not used. Without importing the script as above, the component does not render.  I did change the name of the script file, though, to d3-scale.js for more clarity that it is a library.  Do you have a suggestion as to how to get this to work?  I'll keep it as is for now.

Reply:
Ooooh ok, let me see if i can work it out!